file_handling/test/test_save.py

```diff
@@ -0,0 +1,104 @@
+import os
+import unittest
+import numpy as np
+
+from file_handling.load import load
+from file_handling.save import save_banks, save_file
+from utils import ROOT_DIR
+
+from mantid.simpleapi import mtd, \
+    LoadNexusProcessed, LoadAscii, ConvertToHistogram
+
+
+class TestSave(unittest.TestCase):
+
+    def setUp(self):
+        self.align_and_focus_args = {
```

Comment:
This doesn't need to be `self.` as it is only referenced in the `setUp` function

Reply:
I'll take it out for now. But when we get a test file with events instead of just histograms, I need to cover the line following line with a test: https://github.com/marshallmcdonnell/mantid_total_scattering/blob/850ce85c1c86809d2d8168a2deda6a8bd05d0c85/file_handling/save.py#L58

To do that, I'll need to change the `PreserveEvents` = `True` in `aligh_and_focus_args` and re-load the file. But, I'll save that for later and drop the `self` for now, thanks!